Gruntfile.js

```diff
@@ -2,18 +2,25 @@
 
 module.exports = function (grunt) {
 
-    // Load external grunt task config.
-    grunt.loadTasks('./grunt');
+    if (grunt.option('help')) {
+        require('load-grunt-tasks')(grunt);
+    } else {
+        require('jit-grunt')(grunt, {
+            force: 'grunt-force-task'
+        });
+    }
 
-    grunt.loadNpmTasks('grunt-contrib-jshint');
-    grunt.loadNpmTasks('grunt-jscs');
+    require('time-grunt')(grunt);
 
     grunt.initConfig({
         config: {
+            lib: 'bower_components',
+            modules: 'src',
             files: {
                 lint: [
                     'src/**/*.js'
```

Comment:
It's just another js file, so it should be linted. I've added it to the linters on projects I've been on